agent/util-scripts/gold/pbench-clear-tools/test-66.txt

```diff
@@ -1,19 +1,10 @@
 +++ Running test-66 pbench-clear-tools --group=bad
-	pbench-clear-tools: invalid --group option ("bad"), directory not found: /var/tmp/pbench-test-utils/pbench/tools-v1-bad
-
-usage:
-pbench-clear-tools --name=<tool-name> [--group=<group-name>] [--remotes=<remote-host>[,<remote-host>]]
-
-The following options are available:
-	-g str --group=str, str = the group from which tools should be removed
-		(the default group is 'default')
-	-n str --name=str, str = a specific tool to be removed.
-		If no tool is specified, all tools in the group are removed
-	-r str --remote=str, str = a specific remote on which tools needs to be cleared.
-		If no remote is specified, all the tools on all remotes are removed
-
+	pbench-clear-tools: invalid option ("bad"), directory not found: /var/tmp/pbench-test-utils/pbench/tools-v1-bad
```

Comment:
This should look the same as the original, and emit a usage message.

Reply:
Fixed its using the built click stuff but the affect is the same.

Reply:
OK, if we're being sticklers about output ... we should keep the old `invalid --group option` instead of dropping the `--group`

Reply:
Can we restore the original message?

Reply:
everything line 14 is auto generated by click.

Reply:
Looks like we can add an argument to the `verify_tool_group()` method to provide to parameter name to emit the message, or just assume use of `verify_tool_group` consistently uses the `--group` option.

Reply:
I used a click built for this

Reply:
Isn't this message emitted at [line 111](https://github.com/distributed-system-analysis/pbench/pull/1841/files#diff-5fb10d19d7397575c37cca2715e85dbb8d39e154bcae1b85fc84bb440fd30f1cR111) of `base.py`?  Why can't we just modify that message to say "invalid --group option ..."?